universe/com.goal_smart/manifest.tt

```diff
@@ -46,120 +46,151 @@ class @com.goal_smart
-                        out rank: Number,
-                        out points: Number)
-  #_[canonical="league relegation, league bottoms, relegation teams, relegation sides"]
-  #_[confirmation="relegation teams of $league"]
-  #_[result = ["${rank} . ${team} . Points: ${points}"]]
-  #[poll_interval=3600000ms]
-  #[doc="Get the relegation teams of a league"];
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
 
   monitorable list query teamFixtures(in req team_id: Entity(com.goal_smart:soccer_team) #_[prompt="what team do you want out check out"],
-                        out league : Entity(com.goal_smart:soccer_league),
-                        out team1: Entity(com.goal_smart:soccer_team),
-                        out team2: Entity(com.goal_smart:soccer_team),
-                        out venue: String)
+                        out league : Entity(com.goal_smart:soccer_league)
+                        #_[canonical = {
+                        base = ["league", "division"]
+                        }],
+                        out team1: Entity(com.goal_smart:soccer_team)
+                        #_[canonical = {
+                        base = ["team", "side", "group"]
+                        }],
+                        out team2: Entity(com.goal_smart:soccer_team)
+                        #_[canonical = {
+                        base = ["team", "side", "group"]
+                        }],
+                        out venue: String
+                        #_[canonical = {
+                        base = ["location", "place", "venue"]
+                        }])
   #_[canonical="team fixtures, team matches, team games"]
   #_[confirmation="fixtures of $team_id"]
   #_[result=["${team1} vs ${team2} at ${venue} for the ${league}"]]
   #[poll_interval=3600000ms]
   #[doc="Get the fixtures of the team"];
 
   monitorable list query previousFixtures(in req team_id: Entity(com.goal_smart:soccer_team) #_[prompt="what team do you want out check out"],
-                        out league : Entity(com.goal_smart:soccer_league),
-                        out team1: Entity(com.goal_smart:soccer_team),
-                        out team2: Entity(com.goal_smart:soccer_team),
-                        out score1: Number,
-                        out score2: Number,
-                        out result: String,
-                        out venue: String)
+                        out league : Entity(com.goal_smart:soccer_league)
+                        #_[canonical = {
+                        base = ["league", "division"]
+                        }],
+                        out our_team: Entity(com.goal_smart:soccer_team)
+                        #_[canonical = {
+                        base = ["team", "side", "group"]
+                        }],
+                        out opposition_team: Entity(com.goal_smart:soccer_team)
+                        #_[canonical = {
+                        base = ["team", "side", "group"]
```

Comment:
You cannot use the same canonical form for two different parameters. The `base` canonical form is used to form questions like "what is the X of Y"
so now the question "what is the team of Y" becomes ambiguous, because it could be asking about the parameter `our_team` or the parameter `opposition_team`.

You need to write unambiguous forms, and you also need to write as many other grammatical forms as you can think of. I'm sure you can think of some - at least "against #" is a valid `preposition` form for `opposition_team`.